temporalnexus/operation.go

```diff
@@ -223,7 +229,29 @@ func (o *workflowRunOperation[I, O]) Start(ctx context.Context, input I, options
 	if err != nil {
 		return nil, err
 	}
-	return &nexus.HandlerStartOperationResultAsync{OperationID: handle.ID()}, nil
+
+	// Create the link information about the new workflow and return to the caller.
+	link := &common.Link{
+		Variant: &common.Link_WorkflowEvent_{
+			WorkflowEvent: &common.Link_WorkflowEvent{
+				Namespace:  nctx.Namespace,
+				WorkflowId: handle.ID(),
+				RunId:      handle.RunID(),
+			},
```

Comment:
This is the link from the handler side, and I'm not setting the event_id or anything else at this moment.